cmd/tnf/generate/catalog/catalog.go

```diff
@@ -201,6 +201,9 @@ func outputTestCases() {
 	// we need the list of suite's names
 	suites := getSuitesFromIdentifiers(keys)
 
+	// Sort the list of suite names
+	sort.Strings(suites)
```

Comment:
Shoudn't we also checkin the sorted catalog.md?

Reply:
it's sorted right now. we've been careful about messing it up since the bug was introduced :smile: